### ISSUE TITLE
WEBJS - Issue with retrieving user messages - request screenshot attached

### ISSUE THREAD

**Issue by mehmetcoskun** [allburov/whatsapp-http-api#106]
Hello,

I am having a problem with retrieving user messages in this WAHA WEBJS. I have tried to make a request, but every time I try to retrieve messages I don't see any results. 

i only get this problem with a few users.

I have attached a screenshot of my request for reference. Can someone please help me understand what I might be doing wrong or if there is something in the code that needs to be adjusted? 

Thank you in advance for your help. 

[screenshot](https://ibb.co/dt17dh4)

**Comment by mehmetcoskun** [comment]
And how to get media url? Right now it gives the enc file, is it not possible to convert it?

**Comment by allburov** [comment]
@mehmetcoskun Hi! 
Do use Core or Plus version? The images are supported only in Plus one.
It's weird that it doesn't work in Core one, we'll take a look!

The right way of receiving messages - is to setup webhook and read messages at the time when an account receives it https://waha.devlike.pro/docs/how-to/receive-messages/

**Comment by mehmetcoskun** [comment]
I am using Plus

**Comment by allburov** [comment]
Hi! 
The issue has been fixed in WAHA Plus 2023.6.9, please update the image and try again!
https://waha.devlike.pro/docs/overview/faq/#how-to-update

**Comment by mehmetcoskun** [comment]
```
{
  "statusCode": 500,
  "timestamp": "2023-06-22T13:31:36.032Z",
  "exception": {
    "stack": "Error: Evaluation failed: TypeError: Cannot read properties of undefined (reading 'mediaStage')\n    at __puppeteer_evaluation_script__:6:31\n    at ExecutionContext._evaluateInternal (/app/node_modules/whatsapp-web.js/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:221:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ExecutionContext.evaluate (/app/node_modules/whatsapp-web.js/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:110:16)\n    at async Message.downloadMedia (/app/node_modules/whatsapp-web.js/src/structures/Message.js:393:24)\n    at async WhatsappSessionWebJSPlus.getMessages (/app/dist/core/session.webjs.core.js:131:25)\n    at async /app/node_modules/@nestjs/core/router/router-execution-context.js:46:28\n    at async /app/node_modules/@nestjs/core/router/router-proxy.js:9:17",
    "message": "Evaluation failed: TypeError: Cannot read properties of undefined (reading 'mediaStage')\n    at __puppeteer_evaluation_script__:6:31",
    "name": "Error"
  },
  "request": {
    "path": "/api/messages?chatId=905555555555%40c.us&limit=100&session=default",
    "method": "GET",
    "body": {},
    "query": {
      "chatId": "905555555555@c.us",
      "limit": "100",
      "session": "default"
    }
  },
  "version": {
    "version": "2023.6.9",
    "tier": "PLUS",
    "browser": "/usr/bin/chromium"
  }
}
```

Still get error while getting media
